### PR TITLE
Update battery3

### PR DESCRIPTION
**Pull request by baldrailers**
Add better icon/label for charging state. This should address [#235](https://github.com/regolith-linux/regolith-desktop/issues/235)

### CODE REVIEW

File: scripts/battery3 [hunk 1]
```diff
@@ -51,7 +51,9 @@ if [ -z "$CHARGE_STATE" ]; then
     elif [ "$CHARGE_STATE" == "discharging" ]; then
     CHARGE_ICON=$(xrescat i3xrocks.label.dn D)
```
Comment (kgilmer) [inline scripts/battery3]: If we're going away from the up = charge, down = discharge semantics, we'll need to replace discharging w/ something else to remain consistent.

File: scripts/battery3 [hunk 2]
```diff
@@ -51,7 +51,9 @@ if [ -z "$CHARGE_STATE" ]; then
...
     elif [ "$CHARGE_STATE" == "charging" ]; then
-    CHARGE_ICON=$(xrescat i3xrocks.label.up C)
+    CHARGE_ICON=$(xrescat i3xrocks.label.batterycharging C)
```
Comment (kgilmer) [inline scripts/battery3]: These Xresource keys have underscores.  See https://github.com/regolith-linux/regolith-styles/blob/master/cahuella/typeface#L18-L19

File: scripts/battery3 (hunk 1 above)
Comment (baldrailers) [reply scripts/battery3]: I agree. Right now the change will address the `charging,full-charged` status, and I can't think of any replacement label for `discharging`. So we can put this on hold until we get something that makes more sense than the current label.

File: scripts/battery3 (hunk 2 above)
Comment (baldrailers) [reply scripts/battery3]: Ah, yeah the labels I have was from 1.2 release that's why there were no underscore. I'll update the PR. Thanks!

File: scripts/battery3 (hunk 1 above)
Comment (kgilmer) [reply scripts/battery3]: Do you mean to revert the change on line 54 or are you looking for a replacement for line 52?

**Comment by baldrailers** [comment]
Closing this as the version of the scripts I'm running is base from 1.2

**Pull request closed without merging**